README.md

```diff
@@ -1,5 +1,10 @@
 # QCVIS — Interactive Visualization of Quantum Algorithms
 
+[![License](https://img.shields.io/github/license/fh-igd-iet/qcvis?color=blue&label=License)](https://fh-igd-iet.github.io/qcvis/)
```

Comment:
- The alt-text should match the image `License: EUPL-1.2`
- The link should point to the license, not to the application `https://github.com/fh-igd-iet/qcvis/blob/trunk/LICENSE.txt`